stravalib/model.py

```diff
@@ -997,3 +998,51 @@ class Route(LoadableEntity):
     # timestamp = NOT IMPLEMENTED
     # segments = NOT IMPLEMENTED
 
+
+class Subscription(LoadableEntity):
+    """
+    Represents a Webhook Event Subscription.
+
+    http://strava.github.io/api/partner/v3/events/
+    """
+    OBJECT_TYPE_ACTIVITY = 'activity'
+    ASPECT_TYPE_CREATE = 'create'
+
+    VERIFY_TOKEN_DEFAULT = 'STRAVA'
+
+    application_id = Attribute(unicode, (SUMMARY, DETAILED))
+    object_type = Attribute(unicode, (SUMMARY, DETAILED))
+    aspect_type = Attribute(unicode, (SUMMARY, DETAILED))
+    callback_url = Attribute(unicode, (SUMMARY, DETAILED))
+    created_at = TimestampAttribute((SUMMARY, DETAILED))
+    updated_at = TimestampAttribute((SUMMARY, DETAILED))
+
+
+class SubscriptionCallback(LoadableEntity):
+    """
+    Represents a Webhook Event Subscription Callback.
+    """
+    def __init__(self, *args, **kwargs):
+        """
+        Special/complicated __init__ because of tricky variable names
+        """
+        setattr(self, 'hub.mode', Attribute(unicode, (SUMMARY, DETAILED)))
+        setattr(self, 'hub.verify_token', Attribute(unicode, (SUMMARY, DETAILED)))
+        setattr(self, 'hub.challenge', Attribute(unicode, (SUMMARY, DETAILED)))
```

Comment:
I think that's a great idea, and it should keep the code more consistent. I'll get to it when I have a moment.